dream_layer_backend/install_cli.sh

```
@@ -0,0 +1,37 @@
+#!/bin/bash
+"""
+Installation script for DreamLayer CLI
+"""
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing a linefeed. Double check syntax.

(SC2289)